src/components/ContentGroup/_contentgroup.scss

```diff
@@ -0,0 +1,56 @@
+$eccgui-color-scontentgroup-border-main: rgba($eccgui-color-workspace-text, $eccgui-opacity-muted) !default;
```

Comment:
This file is not currently not included via https://github.com/eccenca/gui-elements/blob/develop/src/components/index.scss